File: src/main/java/com/hackclub/hccore/commands/ShrugCommand.java
package com.hackclub.hccore.commands;

import com.hackclub.hccore.HCCorePlugin;
import org.bukkit.ChatColor;
import org.bukkit.command.Command;
import org.bukkit.command.CommandExecutor;
import org.bukkit.command.CommandSender;
import org.bukkit.entity.Player;

public class ShrugCommand implements CommandExecutor {
    private static final String SHRUG = "¯\\_(ツ)_/¯";

    private final HCCorePlugin plugin;

    public ShrugCommand(HCCorePlugin plugin) {
        this.plugin = plugin;
    }

    @Override
    public boolean onCommand(
        CommandSender sender,
        Command cmd,
        String alias,
        String[] args
    ) {
        if (!(sender instanceof Player)) {
            sender.sendMessage(
                ChatColor.RED + "You must be a player to use this"
            );
            return true;
        }

        Player player = (Player) sender;
        if (args.length == 0) {
            player.chat(ShrugCommand.SHRUG);
        } else {
            player.chat(String.join(" ", args) + " " + ShrugCommand.SHRUG);
        }

        return true;
    }
}

File: src/main/java/com/hackclub/hccore/commands/ShrugCommand.kt
package com.hackclub.hccore.commands

import com.hackclub.hccore.HCCorePlugin
import com.hackclub.hccore.commands.ShrugCommand
import org.bukkit.ChatColor
import org.bukkit.command.Command
import org.bukkit.command.CommandExecutor
import org.bukkit.command.CommandSender
import org.bukkit.entity.Player

class ShrugCommand(private val plugin: HCCorePlugin) : CommandExecutor {
    override fun onCommand(sender: CommandSender, cmd: Command, alias: String, args: Array<String>): Boolean {
        if (sender !is Player) {
            sender.sendMessage(ChatColor.RED.toString() + "You must be a player to use this")
            return true
        }
        val player = sender
        if (args.size == 0) {
            player.chat(SHRUG)
        } else {
            player.chat(java.lang.String.join(" ", *args) + " " + SHRUG)
        }
        return true
    }

    companion object {
        private const val SHRUG = "¯\\_(ツ)_/¯"
    }
}